drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -331,6 +335,54 @@ function DragAndDropTemplates(configuration) {
         );
     };
 
+    var getFixedSizingFeedbackDetailTemplate = function(ctx) {
+        var attempts_used_html = h('div.attempts', [
+            gettext('{used} / {total}').replace('{used}', ctx.attempts).replace('{total}', ctx.max_attempts),
```

Comment:
If `max_attempts` is unset it will show up as `null` here. If `max_attempts` is `null` the message should not display it at all.